untitled/main/com/aca/homework/week11/occurence/StringOfNumberSupplierTest.java

```diff
@@ -0,0 +1,25 @@
+package com.aca.homework.week11.occurence;
+
+public class StringOfNumberSupplierTest implements Supplier<String> {
```

Comment:
please change this class to take a list of strings, it will be more intuitive when writing unit test